### PR TITLE
Add new API V3 connector

### PR DESCRIPTION
**Pull request by FloxoS**
Hello!

We built a new API @Tilkee and we built a new connector based on it.
We will look to deprecate the previous one after this one has been published.

Let me know if I missed something or did something wrong.

Regards,
Florian

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [X] I attest that the connector works and I verified by deploying and testing all the operations. 
- [X] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [X] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

### CODE REVIEW

**Comment by FloxoS** [comment]
Hello @Amjed-Ayoub 

I hope you're doing well.

Do you have any update regarding our PR or an ETA for when you'll process it ?

Thanks for your help.

Regards,
Florian

**Comment by Amjed-Ayoub** [comment]
Hi @FloxoS,  
Thank you for reaching out. Our Certification Engineer will review your request. Please expect to hear back from him soon.
Thank you for your patience.

File: certified-connectors/Tilkee-V3/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Tilkee
```
Comment (sriyen-msft) [inline certified-connectors/Tilkee-V3/README.md:1]: @FloxoS There is a folder called "Tilkee" in the repo, and you are trying to create another one for the same connector which is not allowed. Please update your existing connector files.

File: certified-connectors/Tilkee-V3/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Tilkee
+
+Tilkee fuels your business by empowering your Sales and Marketing teams through reliable and actionable data. This connector offers the ability to integrate your other applications with Tilkee.
+
```
Comment (sriyen-msft) [inline certified-connectors/Tilkee-V3/README.md:4]: @FloxoS Please refer the templates provided for readme.md file in the repo and add required sections.

File: certified-connectors/Tilkee-V3/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,2241 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.3",
+    "description": "Tilkee is an online platform that makes cross-functional collaboration happen. Combining deep knowledge of sales enablement with augmented intelligence, the vision of the team behind Tilkee is to align marketing with sales and guide Sales, Product Marketing, Channel Enablement, and Customer Success teams towards digital transformation.",
+    "title": "Tilkee V3",
```
Comment (sriyen-msft) [inline certified-connectors/Tilkee-V3/apiDefinition.swagger.json:6]: @FloxoS Please revert back the title as it is not allowed to change the title and with version numbers.

File: certified-connectors/Tilkee-V3/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,2241 @@
...
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/v3/oauth/authorize/": {},
```
Comment (sriyen-msft) [inline certified-connectors/Tilkee-V3/apiDefinition.swagger.json:25]: @FloxoS Empty operation definitions can be removed as they would not add any value to your connector.

File: certified-connectors/Tilkee-V3/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,2241 @@
...
+          "application/json"
+        ],
+        "parameters": [],
+        "responses": {
+          "200": {
+            "description": "Successful Response",
+            "schema": {
+              "$ref": "#/definitions/User"
+            },
+            "headers": {}
```
Comment (sriyen-msft) [inline certified-connectors/Tilkee-V3/apiDefinition.swagger.json:55]: @FloxoS Empty properties can be removed until unless you have a specific reason to keep them.

File: certified-connectors/Tilkee-V3/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,2241 @@
...
+          }
+        },
+        "security": [
+          {
+            "oauth2_auth": []
```
Comment (sriyen-msft) [inline certified-connectors/Tilkee-V3/apiDefinition.swagger.json:60]: @FloxoS It is not required to have a "security" section for each operation. please remove it and test your connector thoroughly before pushing the updates to the PR.

File: certified-connectors/Tilkee-V3/apiProperties.json [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+            "refreshUrl": {
+              "value": "https://app.tilkee.com/v3/oauth/authorize/"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#e44719",
+    "capabilities": [],
+    "publisher": "Flo Sanz"
```
Comment (sriyen-msft) [inline certified-connectors/Tilkee-V3/apiProperties.json:34]: @FloxoS It is not allowed to have an individual name as publisher for certified connectors. Please provide your organization or company name.

Comment (sriyen-msft) [inline certified-connectors/Tilkee-V3/apiProperties.json:34]: @FloxoS The stackOwner property is missing which is a required property. 

Docs - [Add Metadata](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-3-add-metadata)

File: certified-connectors/Tilkee-V3/apiProperties.json [hunk 8]
```diff
@@ -0,0 +1,36 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "ToBeGeneratedOnTilkee",
```
Comment (sriyen-msft) [inline certified-connectors/Tilkee-V3/apiProperties.json:8]: @FloxoS Please update it to `"[[TEST]]` to maintain the consistency with other connectors.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by FloxoS** [comment]
Hello @sriyen-msft,

Thanks for your review, I've taken into account all your feedbacks.

I replaced all my files in previous Tilkee folder, however as it is a major update, it now generates the following error: 
`The operation has been deleted. This change breaks existing flows.`

As we don't have any customer using this previous connector in production, that's not an issue for us to break it.

I'm not sure of how we should proceed in such case, can you explain me what to do? 

Thanks for your help!

**Review by v-eisljami — APPROVED** [review]

**Comment by v-eisljami** [comment]
Hey @FloxoS. Hope you are doing well. We noticed that you removed some actions from your connector. Four operations have been deleted. We need to make sure that no clients will be affected by these changes. Can you confirm that no connector users will be affected by these operation removals?

**Comment by FloxoS** [comment]
Hi @v-eisljami.
Thanks for reaching out. I can confirm that we currently don't have any customer using the current PowerAutomate plugin. We are waiting this major update in order to start a new communication about it.

Let me know if you need more information.

Regards,
Florian

**Review by raghavthind2005 — APPROVED** [review]

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**